richtextfx/src/main/java/org/fxmisc/richtext/model/StyledText.java

```diff
@@ -94,15 +114,30 @@ public void encode(DataOutputStream os, StyledText<S> t) throws IOException {
 
     private final String text;
 
+    /**
+     * The text content of this piece of styled text.
+     */
     public String getText() { return text; }
 
     private final S style;
+
+    /**
+     * The style of this piece of styled text.
+     */
     public S getStyle() { return style; }
 
+    /**
+     * Creates a new StyledText with the same content but the given style.
+     *
+     * <p>This class is unmodified.</p>
```

Comment:
Does "This class is unmodified" need to be there since you already specify that the method creates a new object and the class-level javadoc says the object is immutable?